src/pages/[...slug].tsx

```diff
@@ -147,6 +181,41 @@ export const getStaticProps: GetStaticProps<PageProps> = async ({ params }) => {
         throw new Error('Missing slug')
     }
 
+    const topLevelPaths = readdirSync(join(process.cwd(), CONTENT_FOLDER), { withFileTypes: true })
+        .filter(directory => directory.isDirectory())
+        .map(directory => directory.name)
+
+    const leftNav: LeftNav[] = topLevelPaths
+        .map(path => {
+            const pathToIndex = join(process.cwd(), CONTENT_FOLDER, path, 'index.md')
+
+            if (!existsSync(pathToIndex)) {
+                return
+            }
+
+            const contents = readFileSync(pathToIndex, 'utf8')
+            const heading = contents.match(/^# (.*)$/m)?.[1]
```

Comment:
I think we should integrate with the markdown rendering code instead, which already generates a ToC and would be more reliable